Mimiware.ServiceResult/Mimiware.ServiceResult.Tests/ServiceResultTests.cs

```diff
@@ -58,6 +58,18 @@ public void OkCreatedResult()
             Assert.True(result.IsSuccessCode);
         }
 
+        [Fact]
+        public void ServiceErrorResult()
+        {
+            var result = new ServiceResult<object>();
```

Comment:
If you are not testing object, maybe you don't have do include?

Reply:
The non-generic class did not have the Error() method, but I added it now.